src/index.css

```diff
@@ -1,4 +1,67 @@
-h2 {
-  font-size: 32px;
-  color: greenyellow;
-}
\ No newline at end of file
+body {
+  font-family: sans-serif;
+  margin: 0.5rem auto;
+  padding: 0;
+}
+
+h1 {
+  font-size: 3rem;
+  margin-top: 2rem;
+  margin-left: 2rem;
+}
+
+p {
+  margin: 2rem;
+  font-size: 20px;
+}
+
+#task-list {
+  margin: 0.5rem auto;
+  width: 95%;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: flex;
+  align-items: center;
+  background-color: #f5f5f5;
+  border: 1px solid #ddd;
+  padding: 1rem;
+  margin-bottom: 1rem;
+}
+
+input[type="checkbox"] {
+  margin-right: 1rem;
+}
+
+.completed {
+  text-decoration: line-through;
+}
```

Comment:
- [ ] [optional] i suggest you should remove the line-through property in order to match the design given and also because you will be implementing this feature dynamically in the upcoming milestones :+1: